currency/currency.go

```diff
@@ -87,7 +87,7 @@ func ToMinorUnit(currencyCode string, amount float64) (minorUnitAmount uint64, e
 	switch {
 	case amount > 0:
 		factor := getCurrencyFactor(currencyCode)
-		minorUnitAmount = uint64(math.Round(amount * float64(factor)))
+		minorUnitAmount = uint64(math.Round(amount * factor))
 	case amount == 0:
```

Comment:
I think no need since the default value of uint is 0

Reply:
`amount` is `float64` can be <= 0

Reply:
I mean the `case amount == 0`

Reply:
Ah, I got what you meant now.
Let me make it shorter.